audit-service/src/unit-tests/resources/application.yaml

```diff
@@ -1,23 +0,0 @@
-# Copyright 2018-2021 Crown Copyright
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-logging:
-  level:
-    uk.gov.gchq.palisade.service.audit.request: DEBUG
-
-audit:
-  implementations:
-    stroom:
-    logger:
-    simple:
```

Comment:
Needs new line

Reply:
This file should have been deleted, bit of a foul up during the merge.